include/freerdp/settings.h

```diff
@@ -833,290 +857,290 @@ struct rdp_settings
 	UINT64 padding001[16 - 1]; /* 1 */
 
 	/* Core Parameters */
-	ALIGN64 BOOL ServerMode; /* 16 */
-	ALIGN64 UINT32 ShareId; /* 17 */
-	ALIGN64 UINT32 PduSource; /* 18 */
-	ALIGN64 UINT32 ServerPort; /* 19 */
-	ALIGN64 char* ServerHostname; /* 20 */
-	ALIGN64 char* Username; /* 21 */
-	ALIGN64 char* Password; /* 22 */
-	ALIGN64 char* Domain; /* 23 */
-	ALIGN64 char* PasswordHash; /* 24 */
-	ALIGN64 BOOL WaitForOutputBufferFlush; /* 25 */
-	ALIGN64 UINT32 MaxTimeInCheckLoop; /* 26 */
+	ALIGN64 BOOL   ServerMode;               /* 16 */
+	ALIGN64 UINT32 ShareId;                  /* 17 */
+	ALIGN64 UINT32 PduSource;                /* 18 */
+	ALIGN64 UINT32 ServerPort;               /* 19 */
+	ALIGN64 char*  ServerHostname;           /* 20 */
+	ALIGN64 char*  Username;                 /* 21 */
+	ALIGN64 char*  Password;                 /* 22 */
+	ALIGN64 char*  Domain;                   /* 23 */
+	ALIGN64 char*  PasswordHash;             /* 24 */
+	ALIGN64 BOOL   WaitForOutputBufferFlush; /* 25 */
+	ALIGN64 UINT32 MaxTimeInCheckLoop;       /* 26 */
 	UINT64 padding0064[64 - 27]; /* 27 */
 	UINT64 padding0128[128 - 64]; /* 64 */
 
 	/**
-	 * GCC User Data Blocks
-	 */
+        * GCC User Data Blocks
+        */
```

Comment:
@informatimago I guess the same tab issue happened to these comments as well? :)